code/__DEFINES/~skyrat_defines/taur_sprite_defines.dm

```diff
@@ -0,0 +1,2 @@
+#define HAS_TAUR_SPRITE 1
+#define HAS_NO_TAUR_SPRITE 0
```

Comment:
I'm not quite sure why you need two states for this, just use `!HAS_TAUR_SPRITES` if it doesn't have taur sprites, or something.
```suggestion
#define HAS_TAUR_SPRITE TRUE
#define HAS_NO_TAUR_SPRITE FALSE
```

Reply:
Good point.